src/views/Users/Login/ResendConfirmation.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
This HTML structure needs to be reviewed as it breaks the menu of the login button:
![image](https://user-images.githubusercontent.com/6538906/92758589-bb5cc900-f343-11ea-94d9-b3c3636c4798.png)